app-misc/terminal-colors/terminal-colors-3.0.2.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="ncurses"
+
+inherit python-r1
+
+DESCRIPTION="A tool to display color charts for 8, 16, 88, and 256 color terminals"
+HOMEPAGE="http://zhar.net/projects/shell/terminal-colors"
+SRC_URI="https://assets.thamognya.com/gentoo/distfiles/${P}.tar.gz"
```

Comment:
Where'd you get the source code? Why not use the original URL? Or is this your own fork (should be advertised in the HOMEPAGE then)

Reply:
Ah, it is not a fork rather it continues what radhermit did which was to package the script seperately in the previous ebuilds. But unfortunately he or she does not maintain does distfiles any longer hence I had to make the tar.gz. I am currently trying to work on making a ebuild that won't rely on that.